### DISCUSSION TITLE
Mods for SWL/DXing

### DISCUSSION THREAD

**Discussion by RichardVanDeLaarschot** [Tunas1337/UV-K5-Modded-Firmwares#52] [Q&A] [answered]
The UV-K5 after modding 18to1300MHz is also great for listing only for the SWL/DXing community.
However, in many places, it is not allowed to have a TX device without certification.
Could you also provide RX-only mods (transmitting disabled)?
Thanks

**Comment by lohtse** [comment, marked as answer]
> The UV-K5 after modding 18to1300MHz is also great for listing only for the SWL/DXing community.
However, in many places, it is not allowed to have a TX device without certification.
Could you also provide RX-only mods (transmitting disabled)?
Thanks















You can do this using the uvmod kitchen




https://github.com/amnemonic/Quansheng_UV-K5_Firmware/tree/main/uvmod_kitchen

**Reply by RichardVanDeLaarschot** [reply]
Thanks i will look into it

**Reply by lohtse** [reply]
Just read the readme.... when I first saw it I though oh crap... but then just tried and it is as easy as feck...

**Reply by RichardVanDeLaarschot** [reply]
Looks easy only I see that the mod_custom_freq_ranges.py only contains the default ranges.
what would the full working ranges be i need to fill in to get 18-1300Mhz?

**Reply by lohtse** [reply]
it explains here """mod_custom_freq_ranges.py
The purpose of this mod is to give ability to user to select which ranges should be covered by each frequency range of quansheng radio or if to extend them at all. By default, file contains frequency ranges exactly like in stock firmware.

Customization:

# change below sets to new ones, values are in Hz
new_freq_low_limit =   [50_000_000, 108_000_000, 136_000_000, 174_000_000, 350_000_000, 400_000_000, 470_000_000]
new_freq_high_limit =  [76_000_000, 135_999_900, 173_999_900, 349_999_900, 399_999_900, 469_999_900, 600_000_000]
Here you can change low and high limit for each frequency band. The underscore _ symbol is omitted by python interpreter and is added only for better readability. So for example, if you want to fill the gap between 76 and 108Mhz then in second array change first limit from 76_000_000 to 107_999_990 or if you want to extend above 600MHz then change last limit from 600_000_000 to 1300_000_000. Please keep in mind that different ranges are demodulated slightly different inside BK4819 chip, and some ranges have enabled AM demodulation while"""


18   would be 18_000_000

**Reply by RichardVanDeLaarschot** [reply]
Thanks

**Reply by lohtse** [reply]
No problem.... Only way I learned was due to the telegram group and the fantastic members there....  If you have not done already I would suggest joining it buddy...   I learn so much and am learning more by the day...

**Reply by RichardVanDeLaarschot** [reply]
Got it working perfectly :), thanks again
